models/ssh_key_authorized_keys.go

```diff
@@ -0,0 +1,219 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
```

Comment:
SSH related files should also be moved out of models

Reply:
One thing at a time...

Reply:
But you have split them from old file?

Reply:
On the advice from @6543 and to prepare for that move.

I think this is already going to be difficult enough to get reviewed without adding in another refactor but if you'd like it in here I can do that.

The issue is that it'll need to be one of the first things merged in 1.16 if not the first thing as it'll be too easy conflict.

Reply:
I think we can keep ssh related code unchanged in this PR or move them to another PR.

Reply:
The changes that have been made are necessary for this PR.